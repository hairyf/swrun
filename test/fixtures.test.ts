import { execa } from 'execa'
import { describe, expect, it } from 'vitest'

describe('fixtures', async () => {
  it('fibonacci', async () => {
    const { stdout } = await execa('node', [
      './bin/oxrun.js',
      './test/fixtures/fibonacci.ts',
    ])

    expect(stdout).toMatchSnapshot()
  })
})
