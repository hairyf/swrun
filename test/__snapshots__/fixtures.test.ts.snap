// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`fixtures > fibonacci 1`] = `"55"`;
